helm/hpcc/values.schema.json

```diff
@@ -327,6 +365,14 @@
           "description": "which planes (if any) the data is replicated onto (primarily bare metal)",
           "type": "array",
           "items": { "type": "string" }
+        },
+        "isDropZone" : {
+          "description": "true if data plane is dropzone",
+          "type": "boolean"
+        },
+        "umask" : {
```

Comment:
why is a umask this necessary/desirable?
Maybe it is, just trying to understand how this fits in with storage/security and non-posix planes.

Reply:
The create file uses this umask... I'm not sure if it's necessary.  We may be able to remove this later.

Reply:
I think a jira to revisit it would make sense.

Reply:
https://track.hpccsystems.com/browse/HPCC-25579